### PR TITLE
Panoptes-js Package: Bump url-parse

### PR DESCRIPTION
**Pull request by goplayoutside3**
## Packages:
app-content-pages
app-project
lib-classifier
lib-panoptes-js

## Describe your changes:

This PR bumps `url-parse` dependency in lib-panoptes-js to the latest version in response to a [security alert](https://github.com/advisories/GHSA-rqff-837h-mm52). The alert level is relatively low, but this PR is also for adding documentation about publishing to lib-panoptes-js contributing docs.

Once this PR is merged, I will publish the updated package to npm so other repos such as [next-cookie-auth-panoptes](https://github.com/zooniverse/next-cookie-auth-panoptes) can use lib-panoptes-js as a dependency.

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?

## Components
- [ ] Has a storybook story been created or updated?
- [ ] Is the component accessible? 
  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - [ ] Is it passing accessibility checks in the storybook?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?

## Publishing

- [ ] Is the changelog updated?
- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?

## Post-merging

- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
- [ ] Is the new feature working or bug now fixed?
  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
  - [ ] Is this approved by our designer?
- [ ] Is this ready for production deployment?

### CODE REVIEW

File: packages/lib-classifier/package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
   },
   "peerDependencies": {
     "@zooniverse/grommet-theme": "3.x.x",
-    "@zooniverse/panoptes-js": "~0.1.0",
+    "@zooniverse/panoptes-js": "~0.1.2",
```
Comment (shaunanoordin) [inline packages/lib-classifier/package.json:48]: [minor query] Why are some of these dependencies using `~0.1.2` while others use `0.1.2` ?

**Review by shaunanoordin — APPROVED** [review]
## PR Review

Packages: `lib-panoptes-js` (and by extension, `app-project`, `lib-classifier`, `app-content-pages`)

This is a minor dependency bump for lib-panoptes-js. This looks good - I'm able to run a fresh bootstrap and test a random test project on app-project locally with no issues.

I only have a minor query about why some of these dependencies lost the `~` in their version specifications, but this is only out of curiosity.

LGTM 👍

File: packages/lib-classifier/package.json (hunk 1 above)
Comment (shaunanoordin) [reply packages/lib-classifier/package.json:48]: Dangit, my review forgot to also ask why `"url-parse": "1.5.10"` lost the `~` in the version specification.

File: packages/lib-panoptes-js/docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -27,4 +27,4 @@ This project adheres to [Semantic Versioning](http://semver.org/), and follows t
 
  ## Publishing
 
- TODO: write docs about publishing to NPM 
\ No newline at end of file
+Instructions on publishing to npm can be found [here](https://github.com/zooniverse/front-end-monorepo/README.md).
```
Comment (shaunanoordin) [inline packages/lib-panoptes-js/docs/CONTRIBUTING.md:30]: WAIT WAIT sorry I missed this in my earlier review! This URL doesn't work, and I think the correct URL has to be `https://github.com/zooniverse/front-end-monorepo/blob/master/README.md`)

File: packages/lib-classifier/package.json (hunk 1 above)
Comment (eatyourgreens) [reply packages/lib-classifier/package.json:48]: `~0.1.0` includes `0.1.2`. 🤔

File: packages/lib-panoptes-js/package.json [hunk 3]
```diff
@@ -14,7 +14,7 @@
   },
   "dependencies": {
     "superagent": "~7.1.1",
-    "url-parse": "~1.5.1"
+    "url-parse": "1.5.10"
```
Comment (eatyourgreens) [inline packages/lib-panoptes-js/package.json:17]: This pins the library to exactly 1.5.10, preventing it from being patched when new versions of 1.5.x come out. Did you intend to do that?

File: packages/lib-panoptes-js/docs/CONTRIBUTING.md (hunk 2 above)
Comment (eatyourgreens) [reply packages/lib-panoptes-js/docs/CONTRIBUTING.md:30]: The Readme is quite long. A link directly to the Publishing section might be more convenient. 
https://github.com/zooniverse/front-end-monorepo#publishing

File: packages/lib-panoptes-js/package.json (hunk 3 above)
Comment (goplayoutside3) [reply packages/lib-panoptes-js/package.json:17]: Oh! No I did not intend to do that. I'll put the tilde back

File: packages/lib-panoptes-js/docs/CONTRIBUTING.md (hunk 2 above)
Comment (goplayoutside3) [reply packages/lib-panoptes-js/docs/CONTRIBUTING.md:30]: Thanks! I'll update the url

**Pull request merged**